app/experimenter/nimbus-ui/src/components/PageResults/index.test.tsx

```diff
@@ -131,20 +133,16 @@ it("toggles between absolute and relative branch comparisons", async () => {
   mockExperiment = mockExperimentQuery("demo-slug").experiment;
   render(<Subject />);
 
-  expect(screen.getAllByText("-0.46 to 0.51", { exact: false })).toHaveLength(
-    6,
-  );
-  expect(screen.queryByText("88.6%", { exact: false })).not.toBeInTheDocument();
+  const relativeTab = screen.getByText("Relative uplift comparison");
+  const absoluteTab = screen.getByText("Absolute comparison");
 
-  fireEvent.click(screen.getByText("See absolute comparison"));
-  await screen.findByText("88.6%", { exact: false });
-  expect(
-    screen.queryByText("-0.46 to 0.51", { exact: false }),
-  ).not.toBeInTheDocument();
+  // shows relative by default
+  expect(relativeTab).toHaveAttribute("aria-selected", "true");
+  expect(absoluteTab).toHaveAttribute("aria-selected", "false");
+  fireEvent.click(absoluteTab);
 
-  fireEvent.click(screen.getByText("See relative comparison"));
-  await screen.findAllByText("-0.46 to 0.51", { exact: false });
-  expect(screen.queryByText("88.6%", { exact: false })).not.toBeInTheDocument();
```

Comment:
I removed these since 1) the values are being tested individually in the other tables, and 2) this was giving me problems I think because the values are technically in the document as both sets of tables are rendered but one is hidden. `toBeVisible` wasn't being nice either 🤷‍♀️